frontend/src/container/CreateAlertV2/context/index.tsx

```diff
@@ -122,7 +127,28 @@ export function CreateAlertProvider(
 		setThresholdState({
 			type: 'RESET',
 		});
-	}, [alertType]);
+
+		if (thresholdsFromURL) {
+			try {
+				const thresholds = JSON.parse(thresholdsFromURL);
+				setThresholdState({
+					type: 'SET_THRESHOLDS',
+					payload: thresholds,
+				});
+			} catch (error) {
+				console.error('Error parsing thresholds from URL:', error);
+			}
+
+			setEvaluationWindow({
+				type: 'SET_INITIAL_STATE_FOR_METER',
+			});
+
+			setThresholdState({
+				type: 'SET_MATCH_TYPE',
+				payload: AlertThresholdMatchType.IN_TOTAL,
+			});
```

Comment:
Not an issue for now since only meter alerts implement the `thresholds` URL param, but the fall back to meter will be problematic when we implement https://github.com/SigNoz/signoz/issues/5291.